.changelog/unreleased/improvements/1336-remove-faulty-receipt-check-during-recv-packet-validate.md

```diff
@@ -0,0 +1,3 @@
+- [ibc-testkit] Remove redundant path constructions in the implementation of
```

Comment:
It sounds like the `error-handling` branch in not in sync with the `main`. Otherwise, we wouldn't have seen the changes from PR 1338 here.